CMakeLists.txt

```diff
@@ -130,7 +133,7 @@ set_target_properties(datachannel-tests PROPERTIES
 	CXX_STANDARD 17)
 set_target_properties(datachannel-tests PROPERTIES OUTPUT_NAME tests)
 target_include_directories(datachannel-tests PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/src)
-target_link_libraries(datachannel-tests datachannel)
+target_link_libraries(datachannel-tests datachannel nlohmann_json::nlohmann_json)
```

Comment:
This might not be necessary since `nlohmann/json` is header-only.

Reply:
I don't think this lib is used at all, so it can actually be removed